raincloudy/faucet.py

```diff
@@ -227,8 +223,8 @@ def _set_watering_time(self, zoneid, value):
     def watering_time(self):
         """Return watering_time from zone."""
         # zone starts with index 0
-        value = 'zone_{}_watering_time'.format(self.id - 1)
-        return self._lookup_attr(value)
+        index = self.id - 1
+        return self._attributes['rain_delay_mode'][index]['auto_watering_time']
```

Comment:
There doesn't seem to be a global watering time attribute anymore. There _are_ a manual and an auto attribute though. I don't know what this did before but it'd be trivial to grab both and either add them together or return the one that is > 0